src/overcooked_ai_py/mdp/overcooked_mdp.py

```diff
@@ -753,8 +753,8 @@ def __hash__(self):
         )
 
     def __str__(self):
-        return 'Players: {}, Objects: {}, Bonus orders: {} All orders: {} Timestep: {}'.format( 
-            str(self.players), str(list(self.objects.values())), str(self.bonus_orders), str(self.all_orders), str(self.timestep))
+        return 'Players: {}, Objects: {}'.format( 
```

Comment:
I believe bonus orders are still being used by parts of the environment, and we would appreciate if you could allow for this flexibility.